README.md

```diff
@@ -47,16 +47,14 @@ linter after my tests I would do so like this.
 [dogma]: https://github.com/lpil/dogma
 
 ```elixir
-# config/config.exs
+# config/dev.exs
```

Comment:
We can't assume any Elixir knowledge in the basic documentation. This change would be better with an accompanying explanation but I'm unsure if it's better than what we currently have.